test/packages/bad_data_stream_owner/data_stream/test/manifest.yml

```diff
@@ -0,0 +1,5 @@
+title: "Test"
+type: metrics
+owner:
+  github: com/name/invalid
```

Comment:
It would be interesting to add this field also in at least one data stream of the `good_v3` test package. Thereby, it would also be checked that it works as expected.